### PR TITLE
es-de: Bumped the version to 3.1.1

### PR DESCRIPTION
**Pull request by leonstyhre**
Bumped the version to 3.1.1

### CODE REVIEW

File: games-emulation/es-de/es_de-3.1.1.recipe [hunk 1]
```diff
@@ -6,9 +6,9 @@ HOMEPAGE="https://es-de.org"
 COPYRIGHT="2024 Northwestern Software AB"
 LICENSE="MIT"
 REVISION="1"
-srcGitRev="842c9966eb73efb3436a4df2dfdd66063ce7361c"
+srcGitRev="a59b8016be3ccaab0a678a552128d06b32e7dc01"
 SOURCE_URI="https://gitlab.com/es-de/emulationstation-de/-/archive/$srcGitRev/emulationstation-de-$srcGitRev.tar.gz"
-CHECKSUM_SHA256="52c820beddba7e08014f589120c087d6b84b79a88b6213bf13a4e025fd728285"
```
Comment (Begasus) [inline games-emulation/es-de/es_de-3.1.1.recipe:11]: Why not use official release?
https://gitlab.com/es-de/emulationstation-de/-/releases

Comment (Begasus) [reply games-emulation/es-de/es_de-3.1.1.recipe:11]: Has been a while, guess I asked previous round also :) Thanks!

Comment (leonstyhre) [reply games-emulation/es-de/es_de-3.1.1.recipe:11]: Thanks for the merge! Two questions though, although I can now install 3.1.1 via HaikuDepot, the Version column in the HaikuDepot application still shows "3.1.0-1". Does it simply take some extra time for this to get updated, or did I miss to update some information field somewhere?

Second question is about the name displayed in the HaikuDepot application, currently it says "es_de" but I would like it to be "ES-DE Frontend". I see that some other applications have nice display names there but I can't figure out a way to apply it, could you please let me know how to accomplish this?

Thanks!

Comment (leonstyhre) [reply games-emulation/es-de/es_de-3.1.1.recipe:11]: Actually, forget about the first question, restarting the HaikuDepot application a second time made the Version field information update to "3.1.1-1"!

**Pull request merged**